backend/inspirehep/records/api/literature.py

```diff
@@ -288,56 +328,56 @@ def push_authors_phonetic_blocks_to_redis(self):
                     nx=True,
                 )
 
-    def add_files(self, data):
+    @staticmethod
+    def add_files(data):
         if not current_app.config.get("FEATURE_FLAG_ENABLE_FILES", False):
             LOGGER.info("Files are disabled")
             return data
 
         if "deleted" in data and data["deleted"]:
-            LOGGER.info("Record is deleted, skipping files.", uuid=self.id)
+            LOGGER.info("Record is deleted, skipping files.")
             return data
 
         documents = data.pop("documents", [])
         figures = data.pop("figures", [])
 
-        added_documents = self.add_documents(documents)
+        added_documents = LiteratureRecord.add_documents(documents)
```

Comment:
Isn't better to make this `classmethod` and call it like that?
```suggestion
        added_documents = cls.add_documents(documents)
```